.github/workflows/linux.yaml

```diff
@@ -0,0 +1,185 @@
+name: Linux
+
+# To debug a job, add the following step:
+# - name: Debug with tmate
+#   uses: mxschmitt/action-tmate@v3
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/macos.yaml'
+
+env:
+  COZY_DESKTOP_DIR: "${{ github.workspace }}/cozy-desktop"
+  COZY_STACK_STORAGE: "${{ github.workspace }}/storage"
+  COZY_URL: "http://localhost:8080"
+  COZY_PASSPHRASE: "cozy"
+  GOPROXY: "https://proxy.golang.org"
+  GO111MODULE: "on"
+  NODE_ENV: "test"
+  COZY_DESKTOP_HEARTBEAT: ${{ 1000 }}
```

Comment:
Why `${{ 1000 }}` and not just `"1000"`?

Reply:
I used this syntax when I thought there might be a difference between string and number variables in GA. I should clean that up indeed.